README.md

```diff
@@ -11,6 +11,11 @@ To run tests:
 yarn test
 ```
 
+To run ruling:
```

Comment:
@stas-vilchik I would be nice to have some description for "ruling", what it's supposed to test